airbyte-cdk/bulk/core/load/src/main/kotlin/io/airbyte/cdk/load/task/DestinationTaskExceptionHandler.kt

```diff
@@ -118,8 +116,6 @@ T : ScopedTask {
                         "Task $innerTask run after its stream ${stream.descriptor} has succeeded. This should not happen."
                     )
                 }
-                log.info { "Stream task $innerTask skipped because stream has already failed." }
```

Comment:
Minor nit, but maybe log tasks that are run after failure?

Reply:
done https://github.com/airbytehq/airbyte/pull/48139/commits/1e19e72f9a50a0874e6290c44b935ad90f52a6d1